schema.sql

```diff
@@ -4,5 +4,6 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INTEGER,
     neutered BOOLEAN,
-    weight_kg REAL
+    weight_kg REAL,
+    species TEXT,
```

Comment:
- [ ] I think this is not the right way to add a colmun you should use the alter command so you can add a column to an existed table so kindly use the alter command for adding a new column. ✅🙏🏻

Reply:
Thank you for your observations, I just added the code to ALTER the table in the schema. I was unsure of what I needed to add there.